backend/src/api/auth/auth.routers.js

```diff
@@ -28,4 +30,28 @@ authRouter.post(
   catchAsync(AuthController.loginUser),
 );
 
+authRouter.get(
+  '/google',
+  passport.authenticate('google', { scope: ['email', 'profile'] }),
+);
+
+authRouter.get(
+  '/google/callback',
+
+  passport.authenticate('google', { session: false }),
+  (req, res, next) => {
+    const sessionToken = jwt.sign(
```

Comment:
Third, you need to save token into user document. Just like with regular sign-in

Reply:
ok